### ISSUE TITLE
Allow declaratively specifying orgzly searches in org file

### ISSUE THREAD

**Issue by ParetoOptimalDev** [orgzly-revived/orgzly-android-revived#85]
- I have searched for existing issues that may be the same as or related to mine.

My thought is that orgzly would look for a file, perhaps named `.orgzly-searches.org` that looks like:

```org
* Agenda
#+begin_example
.it.done ad.7
#+end_example
```

Or if some find that too noisy and don't also use mixed pitch fonts:

```org
* Agenda
.it.done ad.7

Maybe both would be pretty easy to support.

If this file exists orgzly would use this as the source of truth for searches.

As I write this, this approach could even be used to declaratively specify other orgzly settings too.

**Comment by amberin** [comment]
Depending on how you look at it, this could be considered solved by the export/import feature added in #526.

**Comment by chaoflow** [comment]
> Depending on how you look at it, this could be considered solved by the export/import feature added in [#526](https://github.com/orgzly-revived/orgzly-android-revived/pull/526).

I agree. How about we pretty-print it (sort keys, indent 2) and place it into a json source block?

**Comment by amberin** [comment]
> > Depending on how you look at it, this could be considered solved by the export/import feature added in [#526](https://github.com/orgzly-revived/orgzly-android-revived/pull/526).
> 
> I agree. How about we pretty-print it (sort keys, indent 2) and place it into a json source block?

Sure, we could do that. But it would require more code when importing settings, and encouraging user fiddling increases the risk of the JSON being invalid when importing, requiring more sophisticated error handling. Maybe I don't fully understand the suggested use case for "declaratively" specifying searches. Why not just tweak things in the app, then export? And keep it clear that the JSON is supposed to be written and read by the app? The only way to use the JSON is to import all settings, anyway, so you'd want to be sure everything is up to date before importing.

**Comment by chaoflow** [comment]
Personally, I find it more convenient to change a text file (json or other) than navigating through an app.
